crates/host/src/wasmbus/mod.rs

```diff
@@ -264,12 +266,39 @@ impl Invocation {
     /// of preventing them from being forged over the network when connected to a lattice,
     /// so an invocation requires a reference to the host (signing) key
     pub fn new(
+        interfaces: &HashMap<String, HashMap<String, WasmCloudEntity>>,
         hostkey: &KeyPair,
         origin: WasmCloudEntity,
-        target: WasmCloudEntity,
-        operation: String,
+        target: Option<&TargetEntity>,
+        operation: impl Into<String>,
         msg: Vec<u8>,
     ) -> anyhow::Result<Invocation> {
+        let operation = operation.into();
+        let (package, operation) = operation
+            .split_once('/')
+            .context("failed to parse operation")?;
+        let target = match target {
+            None => interfaces
+                .get(package)
+                .and_then(|links| links.get("default"))
```

Comment:
Consider a named constant, to disambiguate among references to default link names and default lattice prefixes

Reply:
I don't really see what's there to disambiguate, `links` is keyed by link names, not lattice prefixes

Reply:
What I mean is distinguishing between semantics of a common string (`"default"`) in the repo. Right now, if someone wants to find all the places we fall back on `default` for a link name, they have to search for this string, and then go through the mental effort of excluding uses where `default` was being used in the context of a lattice prefix. (I had to go through this effort a week ago when adding support for custom lattice prefixes and custom link names)

Swapping to named constants like `DEFAULT_LINK_NAME` and `DEFAULT_LATTICE_PREFIX` would remove any ambiguity